interview-questions.md

```diff
@@ -8,50 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: A parameter is a place holder that a function uses for any given argument. The argument is the data itself being fed to the function.
```

Comment:
Great definition.  To help expand this, try providing an example to demonstrate the concept.